packages/disputer/index.js

```diff
@@ -155,12 +155,8 @@ async function run(
       await delay(Number(pollingDelay));
     }
   } catch (error) {
-    logger.error({
-      at: "Disputer#index",
-      message: "Disputer polling error🚨",
-      error: typeof error === "string" ? new Error(error) : error
-    });
-    await waitForLogger(logger);
+    // If any error is thrown, catch it and bubble up to the main try-catch for error processing in the Poll function.
```

Comment:
This change addressed the problematic error handling between the try-catch block.

Reply:
It can be ignored and should be reviewed in PR [1916](https://github.com/UMAprotocol/protocol/pull/1916)